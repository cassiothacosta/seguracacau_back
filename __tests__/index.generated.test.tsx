import renderer from 'react-test-renderer'
import Head from 'next/head';
import styles from '../styles/Home.module.css';
import Home from '../src/pages/index';

jest.mock('next/head');
jest.mock('../styles/Home.module.css');

const renderTree = (tree: any) => renderer.create(tree);
describe('<Home>', () => {
  it('should render component', () => {
    expect(renderTree(<Home 
    />).toJSON()).toMatchSnapshot();
  });
  
});