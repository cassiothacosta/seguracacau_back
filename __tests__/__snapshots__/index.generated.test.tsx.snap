// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> should render component 1`] = `
<div
  className="jsx-e23c2f6fc798f3bf "
>
  <main
    className="jsx-e23c2f6fc798f3bf"
  >
    <h1
      className="jsx-e23c2f6fc798f3bf "
    >
      Welcome to 
      <a
        className="jsx-e23c2f6fc798f3bf"
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
    <p
      className="jsx-e23c2f6fc798f3bf "
    >
      Get started by editing 
      <code
        className="jsx-e23c2f6fc798f3bf"
      >
        pages/index.js
      </code>
    </p>
    <div
      className="jsx-e23c2f6fc798f3bf "
    >
      <a
        className="jsx-e23c2f6fc798f3bf "
        href="https://nextjs.org/docs"
      >
        <h3
          className="jsx-e23c2f6fc798f3bf"
        >
          Documentation →
        </h3>
        <p
          className="jsx-e23c2f6fc798f3bf"
        >
          Find in-depth information about Next.js features and API.
        </p>
      </a>
      <a
        className="jsx-e23c2f6fc798f3bf "
        href="https://nextjs.org/learn"
      >
        <h3
          className="jsx-e23c2f6fc798f3bf"
        >
          Learn →
        </h3>
        <p
          className="jsx-e23c2f6fc798f3bf"
        >
          Learn about Next.js in an interactive course with quizzes!
        </p>
      </a>
      <a
        className="jsx-e23c2f6fc798f3bf "
        href="https://github.com/vercel/next.js/tree/canary/examples"
      >
        <h3
          className="jsx-e23c2f6fc798f3bf"
        >
          Examples →
        </h3>
        <p
          className="jsx-e23c2f6fc798f3bf"
        >
          Discover and deploy boilerplate example Next.js projects.
        </p>
      </a>
      <a
        className="jsx-e23c2f6fc798f3bf "
        href="https://vercel.com/import?filter=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      >
        <h3
          className="jsx-e23c2f6fc798f3bf"
        >
          Deploy →
        </h3>
        <p
          className="jsx-e23c2f6fc798f3bf"
        >
          Instantly deploy your Next.js site to a public URL with Vercel.
        </p>
      </a>
    </div>
  </main>
  <footer
    className="jsx-e23c2f6fc798f3bf"
  >
    <a
      className="jsx-e23c2f6fc798f3bf"
      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      Powered by
       
      <img
        alt="Vercel"
        className="jsx-e23c2f6fc798f3bf "
        src="/vercel.svg"
      />
    </a>
  </footer>
</div>
`;
